ability/ability/Ability.cpp

```diff
@@ -194,7 +194,6 @@ std::vector<kitten::K_GameObject*> ability::Ability::getCardsInHand()
 void ability::Ability::triggerTPEvent(ability::TimePointEvent::TPEventType p_tp, unit::Unit * p_target, AbilityInfoPackage * p_info)
 {
 	unit::StatusContainer* sc = p_target->getStatusContainer();
-	if (sc == nullptr) return;
```

Comment:
Is this change intentional? Not sure what the effects on abilities are

Reply:
O, donno, it sometimes crashed when it was going through because that was null. 
put the check and forgot about it, dont know if that would cause problems or fix any.

Reply:
because what was null ?

Reply:
oh this update removes it. 
My bad, I put the sc there in the original AI code, i don't know why this one removes it.